server/src/server.ts

```diff
@@ -180,6 +184,14 @@ export default function startServer(): void {
     app.put('/api/v2/organizations/:id', userIsAuthorized, userIsAdmin, (req, res) => updateOrganization(req, res));
     app.delete('/api/v2/organizations', userIsAuthorized, userIsAdmin, (req, res) => deleteOrganization(req, res));
 
+    /**
+     * Direct Debit Mandate
+     */
+    app.get('/api/v2/directDebitMandate', userIsAuthorized, (req, res) => getDirectDebitMandate(req, res));
+    app.get('/api/v2/directDebitMandates', userIsAuthorized, userIsAdmin, (req, res) => getDirectDebitMandates(req, res));
+    app.post('/api/v2/directDebitMandate', userIsAuthorized, (req, res) => createDirectDebitMandate(req, res));
+    app.delete('/api/v2/directDebitMandate', userIsAuthorized, (req, res) => deleteDirectDebitMandate(req, res));
```

Comment:
Urls immer plural, und kebab-case

```suggestion
    app.get('/api/v2/direct-debit-mandates', userIsAuthorized, (req, res) => getDirectDebitMandate(req, res));
    app.get('/api/v2/direct-debit-mandates', userIsAuthorized, userIsAdmin, (req, res) => getDirectDebitMandates(req, res));
    app.post('/api/v2/direct-debit-mandates', userIsAuthorized, (req, res) => createDirectDebitMandate(req, res));
    app.delete('/api/v2/direct-debit-mandates', userIsAuthorized, (req, res) => deleteDirectDebitMandate(req, res));
```

Um jetzt in z. 190 noch unterscheiden zu können, was was ist bitte das unter den user hängen 

/api/v2/users/:id/direct-debit-mandates
Dann muss man auch nicht per body das mitsenden